Automattic/jetpack
# Fix/table interstitial loading buttons

## Proposed changes:

This PR updates the table interstitial screens in My Jetpack so that only one button shows in a "loading" state after being clicked instead of both buttons showing the loading icon (which is what happens now)

I chose the first approach that came to mind - if you spot another solution that would be better, let me know!

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Get a test site w/ Jetpack Beta - don't apply this PR yet to see the original issue
* Navigate to `/wp-admin/admin.php?page=my-jetpack#/add-boost`
* Click on "Start for free"
* Notice that both buttons show the loading spinner
![Screenshot 2024-03-26 at 5 24 07 PM](https://github.com/Automattic/jetpack/assets/18016357/b0ffa14f-47ed-4a10-a529-43fb7042614c)
* Now, with this PR applied, visit `/wp-admin/admin.php?page=my-jetpack#/add-boost` again
* This time, when you click "Start for free", only the button you clicked on should show the loading spinner
![Screenshot 2024-03-26 at 5 29 39 PM](https://github.com/Automattic/jetpack/assets/18016357/296f309b-9313-4797-99a1-1e95ad95cac0)
